### PR TITLE
rnpm support for iOS added

### PR DESCRIPTION
**Pull request by 7ynk3r**
I've created a project that contains the m/h files.

### CODE REVIEW

**Comment by joeferraro** [comment]
@7ynk3r cheers for this. can you update the README accordingly?

**Comment by 7ynk3r** [comment]
README updated

File: README.md [hunk 1]
```diff
@@ -4,17 +4,8 @@ Cookie manager for react native.
...
-1. `npm install react-native-cookies`
-2. In the XCode's "Project navigator", right click on project name folder ➜ `Add Files to <...>`
- - Ensure `Copy items if needed` and `Create groups` are checked
-3. Go to `node_modules` ➜ `react-native-cookies` ➜ add `RNCookieManagerIOS` folder
-4. Compile and have some cookies
-
-#### android (only tested on react-native <= 0.18.1)
 1. `npm install react-native-cookies`
 2. `rnpm link react-native-cookies` - (run `npm install -g rnpm` if required)
-3. Om nom nom nom cookies.
```
Comment (Psykar) [inline README.md]: No one likes my humour :(

Comment (7ynk3r) [inline README.md]: humour is back ;)

**Comment by cmcewen** [comment]
would love to see this get merged + released!

**Comment by bgeihsgt** [comment]
:+1:

**Comment by Psykar** [comment]
Those are some truly horrible commit messages 👎 
But the changes themselves seem OK (without having tested)

**Pull request merged**